Stu Card updates (#54)
ietf-wg-drip/draft-ietf-drip-arch: draft-ietf-drip-arch.md

> * Methods for instantly establishing secure communications between an Observer and the pilot of an observed UAS ({{dripcontact}}), using the DRIP support for dynamic contact required by GEN-4 {{RFC9153}}.

> * Privacy in UAS RID messages (PII protection) ({{privacyforbrid}}).
>

> > This document should serve as a single point of entry into the set of IETF documents addressing the basic DRIP requirements.

Reviewer: ```suggestion
> This document should serve as a main point of entry into the set of IETF documents addressing the basic DRIP requirements.
```

This document uses terms defined in {{RFC9153}}.

> Some of the acronyms have plural forms that remain the same as their singular forms, e.g., UAS can expand to Unmanned Aircraft System (singular) or Unmanned Aircraft Systems (plural), as appropriate for the context. 

Reviewer: ```suggestion
Some of the acronyms have plural forms that remain the same as their singular forms, e.g., UAS can expand to Unmanned Aircraft System (singular) or Unmanned Aircraft Systems (plural), as appropriate for the context.  This usage is consistent with Section 2.2 of {{RFC9153}}.
```

DRIP Identity Management Entity (DIME):

> > An entity that performs functions similar to a domain registrar/registry. A DIME vets Claims and/or Evidence from a registrant and delivers back Endorsements and/or Certificates in response. It is a high-level entity in the DRIP registration/provisioning process that can hold the role of HDA, RAA or root of trust (likely the HHIT prefix owner and/or DNS apex owner) for DETs.

Reviewer: ```suggestion
> An entity that performs functions similar to a domain registrar/registry. A DIME vets Claims and/or Evidence from a registrant and delivers back Endorsements and/or Certificates in response. It is a high-level entity in the DRIP registration/provisioning process that can hold the role of HDA, RAA, or root of trust (typically the HHIT prefix owner or DNS apex owner) for DETs.
```

Reviewer 2: In this case, it is not an example, but a clarification that the root _of trust_ for any HHIT hierarchy would naturally be the root _of that hierarchy_ in address space (the IPv6 prefix owner) or name space (the DNS apex owner), and further that those 2 roots would typically be the same. Perhaps rather than "likely" we should use another word, such as "typically"?

Reviewer: "Typically" is better, thanks.

Self-asserting in this usage means that given the Host Identity (HI), the HHIT ORCHID construction (see section 3.5 of {{I-D.ietf-drip-rid}}) and a signature of the DIME on the HHIT and HI; the HHIT can be verified by the receiver as a trusted UAS ID. The explicit registration hierarchy within the HHIT provides registration discovery (managed by a DRIP Identity Management Entity (DIME)) to either yield the HI for a 3rd-party (seeking UAS ID endorsement) validation or prove that the HHIT and HI have been registered uniquely.

## UAS Remote Identifiers Problem Space ##

A DRIP entity identifier needs to be "Trustworthy" (see DRIP Requirement GEN-1, ID-4 and ID-5 in {{RFC9153}}). This means that given a sufficient collection of UAS RID messages, an Observer can establish that the identifier claimed therein uniquely belongs to the claimant. To satisfy DRIP requirements and maintain important security properties, the DRIP identifier should be self-generated by the entity it names (e.g., a UAS) and registered (e.g., with a USS, see Requirements GEN-3 and ID-2).

> However Broadcast RID, especially its support for Bluetooth 4, imposes severe constraints. The previous revision of ASTM UAS RID, F3411-19, allowed a UAS ID of types 1, 2 and 3, each of 20 bytes. {{F3411-22a}} added type 4, Specific Session ID, for other standards development organizations (SDOs) to extend ASTM UAS RID. Type 4 uses one byte to index the Specific Session ID sub-type, leaving 19 bytes (see ID-1 of DRIP Requirements {{RFC9153}}). ASTM has allocated Specific Session ID sub-type 1 to IETF DRIP.

Reviewer: ```suggestion
However Broadcast RID, especially its support for Bluetooth 4, imposes severe constraints. A previous revision of the ASTM UAS RID, F3411-19, allowed a UAS ID of types (1, 2, and 3), each of 20 bytes. {{F3411-22a}} adds type 4, Specific Session ID, for other Standards Development Organizations (SDOs) to extend ASTM UAS RID. Type 4 uses one byte to index the Specific Session ID subtype, leaving 19 bytes (see ID-1 of DRIP Requirements {{RFC9153}}). As described in Section 3 of {{RFC9153}}, ASTM has allocated Specific Session ID subtype 1 to IETF DRIP.
```

Reviewer: align the wording with what we already have in 9153 + back the statements

> The maximum ASTM UAS RID Authentication Message payload is 201 bytes each for authentication types 1, 2, 3 and 4. {{F3411-22a}} added type 5 for IETF and other SDOs to extend ASTM UAS RID with Specific Authentication Methods (SAM). With type 5, one of the 201 bytes is consumed to index the sub-type, leaving only 200 bytes for DRIP authentication payloads, including one or more DRIP entity identifiers and associated authentication data.

Reviewer: ```suggestion
The maximum ASTM UAS RID Authentication Message payload is 201 bytes each for Authentication Types 1, 2, 3, and 4. {{F3411-22a}} adds Authentication Type 5 for other SDOs (including the IETF) to extend ASTM UAS RID with Specific Authentication Methods (SAM). With type 5, one of the 201 bytes is consumed to index the SAM Type, leaving only 200 bytes for DRIP authentication payloads, including one or more DRIP entity identifiers and associated authentication data.
```

The severe constraints of Broadcast RID make it challenging to satisfy UAS RID requirements. From received Broadcast RID messages and information that can be looked up using the received UAS ID in online registries or local caches, it is possible to establish levels of trust in the asserted information and the Operator.

A combination of different DRIP Authentication Messages enables an Observer, without Internet connection (offline) or with (online), to validate a UAS DRIP ID in real-time.  Some messages must contain the relevant registration of the UA's DRIP ID in the claimed DIME.  Some messages must contain sender signatures over both static (e.g., registration) and dynamically changing (e.g., current UA location) data.  Combining these two sets of information, an Observer can piece together a chain of trust including real-time evidence to make a determination on the UA's claims.

> This process (combining the DRIP entity identifier, registries and authentication formats for Broadcast RID) can satisfy the following DRIP requirements defined in {{RFC9153}}: GEN-1, GEN-2, GEN-3, ID-2, ID-3, ID-4 and ID-5.

Reviewer: ```suggestion
This process (combining the DRIP entity identifier, registries, and authentication formats for Broadcast RID) can satisfy the following DRIP requirements defined in {{RFC9153}}: GEN-1, GEN-2, GEN-3, ID-2, ID-3, ID-4, and ID-5.
```
